manager/backingimage.go

```diff
@@ -67,20 +67,13 @@ func (m *VolumeManager) GetBackingImageDataSourcePod(name string) (*v1.Pod, erro
 	return pod, nil
 }
 
-func (m *VolumeManager) GetDefaultBackingImageManagersByDiskUUID(diskUUID string) (*longhorn.BackingImageManager, error) {
-	defaultImage, err := m.ds.GetSettingValueExisted(types.SettingNameDefaultBackingImageManagerImage)
-	if err != nil {
-		return nil, err
-	}
-
+func (m *VolumeManager) GetBackingImageManagersByDiskUUID(diskUUID string) (*longhorn.BackingImageManager, error) {
 	bims, err := m.ds.ListBackingImageManagersByDiskUUID(diskUUID)
 	if err != nil {
 		return nil, err
 	}
 	for _, bim := range bims {
-		if bim.Spec.Image == defaultImage {
-			return bim, nil
-		}
+		return bim, nil
```

Comment:
Do we still need the below code? It think it's still required to prevent getting the old bim before it gets deleted.

```go
		if bim.Spec.Image == defaultImage {
			return bim, nil
		}
```

Reply:
Kept the check statement. Thanks